crates/client/metrics/src/block_metrics.rs

```diff
@@ -1,4 +1,6 @@
-use dc_metrics::{Gauge, MetricsRegistry, PrometheusError, F64};
+// use dc_metrics::{Gauge, MetricsRegistry, PrometheusError, F64};
+
+use crate::{Gauge, MetricsRegistry, PrometheusError, F64};
 
 #[derive(Clone, Debug)]
 pub struct BlockMetrics {
```

Comment:
I would prefer if you kept BlockMetrics where it was in sync, but move the l1 stuff to a new struct L1Metrics inside of the eth crate

i think each service having its own metrics struct for the metrics it expose is better

Reply:
it was creating a circular dependency, `sync` crate is calling the `sync` function from the `eth/l1` crate and `eth` crate needed the `BlockMetrics`.

Reply:
@cchudant added l1BlockMetrics and added it in the client.rs inside the eth crate as of now, it would also have the l1_block_price, for the price from the l1